ts/enums.ts

```diff
@@ -0,0 +1,5 @@
+export enum ButtonAction {
+    Today,
+    PreviousWeek,
+    NextWeek
+  }
```

Comment:
in such a small project I think we can have just one types.ts file, no need to have enums.ts and models.ts files

I have never actually came across naming files with types 'models' in JS. we usually have types.ts or sometimes constants.ts for enums